<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSlider" name="HeightOfJumpSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to game</string>
   </property>
  </widget>
  <widget class="QSlider" name="ColumnPaceSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="PaceOfFallSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="BirdSkinGroupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>411</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Bird's skin </string>
   </property>
   <widget class="QListView" name="BirdSkinListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ChooseBirdPictureButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose picture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearBirdListButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetBirdSkinButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set skin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ColumnPicturesGroupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>411</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Column pictures ( first - lower, second - upper)</string>
   </property>
   <widget class="QPushButton" name="ChangeColumnPicturesButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set pictures</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearColumnPicturesButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear all</string>
    </property>
   </widget>
   <widget class="QListView" name="ColumnPicturesListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ChooseColumnPicturesButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose picture</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="BackgroundGroupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>411</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Background image</string>
   </property>
   <widget class="QPushButton" name="SetBackgroundButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set background</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearBackgroundButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QListView" name="BackgroundListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ChooseBackgroundButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose picture</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
